Added doc for privacy policy URL setting

imankur opened the pull request:
Added doc for privacy policy URL setting

tejas-haptik reviewed (changes requested):
We need to add this in `Customisation` page as well right?
The place where we have added it currently(initialisation through code) is used very less by clients. @imankur

imankur commented:
> We need to add this in `Customisation` page as well right?
> The place where we have added it currently(initialisation through code) is used very less by clients. @imankur

Good point. Added.

tejas-haptik reviewed (approved):
Looks good 👍
